### PR TITLE
Task 1 Solution

### PR DESCRIPTION
**Pull request by uglyprincess**
__1__ 

1

#### Short description of what this resolves:
I have attempted to achieve the purpose of task 1; however, I am facing certain difficulties. Kindly look into the code.

#### Changes proposed in this pull request and/or Screenshots of changes:

-
-
-

### CODE REVIEW

**Comment by deepakDOTexe** [comment]
Either use csv and json modules or pandas not both. You are converting the dataframe generated from csv file to json and then again using json_dump method which is not required. 
If you use pandas then you might find it difficult to generate json with only comedy movies, so I would recommend you to use the csv and json modules only.

**Comment by uglyprincess** [comment]
Okay sir. Thank you.

**Comment by uglyprincess** [comment]
I've made the necessary changes, sir.

File: Submissions/uglyprincess/task_1/task1.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import csv, json
+
+csvFilePath = "C:/Users/Aaryan/Desktop/Monty_Python_Class/Task_1/movies.csv"
```
Comment (Lokesh2703) [inline Submissions/uglyprincess/task_1/task1.py]: Try to use relative path to the file which we gave.

Comment (Lokesh2703) [reply Submissions/uglyprincess/task_1/task1.py]: like "../../../TAsk_folder".

Comment (uglyprincess) [reply Submissions/uglyprincess/task_1/task1.py]: Okay sir

**Comment by uglyprincess** [comment]
I've made the changes sir.

**Comment by uglyprincess** [comment]
This should (probably) do it.

**Pull request merged**